common/src/main/resources/movies.cypher

```diff
@@ -1,11 +1,11 @@
 CREATE (TheMatrix:Movie {title:'The Matrix', released:1999, tagline:'Welcome to the Real World'})
-CREATE (Keanu:Person {name:'Keanu Reeves', born:1964})
-CREATE (Carrie:Person {name:'Carrie-Anne Moss', born:1967})
-CREATE (Laurence:Person {name:'Laurence Fishburne', born:1961})
-CREATE (Hugo:Person {name:'Hugo Weaving', born:1960})
-CREATE (AndyW:Person {name:'Andy Wachowski', born:1967})
-CREATE (LanaW:Person {name:'Lana Wachowski', born:1965})
-CREATE (JoelS:Person {name:'Joel Silver', born:1952})
+CREATE (Keanu:Person:Other {name:'Keanu Reeves', id: randomUUID(), born:1964})
```

Comment:
This is the same file used for apoc.examples.movies? we should perhaps not change it if it is as it would be a breaking change I imagine, perhaps make a new file for testing purposes instead

Reply:
Oh, i didn't noticed it :( Now I created another file